x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -437,6 +437,34 @@ This is only supported with 3rd party S3 providers.  AWS does not support path s
 In order to make AWS API calls, `aws-s3` input requires AWS credentials. Please
 see <<aws-credentials-config,AWS credentials options>> for more details.
 
+[float]
+==== `backup_to_bucket_arn`
```

Comment:
Do we need to add `s3:PutObject` into the AWS Permissions section of this documentation at line 469?

Reply:
good catch! also `s3:DeleteObject`